cmd/registry/config.yml

```diff
@@ -4,6 +4,20 @@ log:
   fields:
     service: registry
     environment: development
+  hooks:
+    type: none # set to "email", and allows user to send email 
+               # only when email parameters is available.
+    levels: 
+      - panic
+    mail:
+     smtp_host: mail.example.com
+     smtp_port: 25
```

Comment:
Given that smtp.(*SendMail) takes an address which is a concatenation of host and port you could just replace smtp_host and smtp_port with smtp_addr and pass the value straight in.